src/containers/_RegFormContainer.jsx

```diff
@@ -53,33 +56,37 @@ const RegFormContainer = () => {
 						Just a few more steps and you`re done! We hate paperwork, too.
 					</RegForm.SubTitle>
 
-					<RegForm.Frame onClick={ () => setEmailFocus(true) } >
-						<RegForm.Label
-							focus={ emailFocus ? true : false }
-						>
-							Email
-						</RegForm.Label>
+					<RegForm.Frame>
 						<RegForm.Input
 							type="text"
+							id="email"
+							placeholder=" "
 							value={ email }
 							onChange={ ({ target }) => setEmail(target.value) }
 							error={ emailError ? true : false }
 						/>
-						{ emailError && <p>Email is required!</p> }
-					</RegForm.Frame>
-
-					<RegForm.Frame onClick={ () => setPasswordFocus(true) } >
 						<RegForm.Label
-							focus={ passwordFocus ? true : false }
```

Comment:
I like how the ternary makes it explicit for true/false
the other can be just `passwordFocus ` (implicitly true/false)